src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -206,15 +206,19 @@ def kubernetes_exception_handler(ex, fault_type, summary, error_message='Error o
         status_code = ex.status
         if status_code == 403:
             logger.warning(message_for_unauthorized_request)
-        if status_code == 404:
+        elif status_code == 404:
```

Comment:
Add one more condition for checking 5xx response codes and classify that as AzureInternalError